lib/msal-react/docs/getting-started.md

```diff
@@ -226,3 +226,35 @@ export function App() {
     }
 }
 ```
+
+## Acquiring an access token outside of a React component
+
+If you require an access token outside of a React component you can directly call the `acquireTokenSilent` function on the `PublicClientApplication`.
+The `PublicClientApplication` can be reused but might as well be recreated for every API call. 
+We do not recommend calling functions that change the user's state outside of a component because those functions rely on the React Context API to work properly.
+
+Keep in mind that the user has to be signed in before you can acquire a token outside of a React component.
+
+```javascript
+import { Configuration, PublicClientApplication } from '@azure/msal-browser'
+
+const acquireAccessToken = async (): Promise<string> => {
```

Comment:
With regards to my comment above, documentation should demonstrate passing an existing pca instance to the function rather than instantiating a new one
```suggestion
const acquireAccessToken = async (pca: PublicClientApplication): Promise<string> => {
```